cpp/cgi_progs/translator.cc

```diff
@@ -131,7 +132,7 @@ std::string CreateEditableRowEntry(const std::string &token, const std::string &
 
 
 void GetDisplayLanguages(std::vector<std::string> *const display_languages, const std::vector<std::string> &translation_languages,
-                         const std::vector<std::string> &additional_view_languages, enum Category category = KEYWORDS) {
+                         const std::vector<std::string> &additional_view_languages, enum Category category, const bool show_macs_col) {
```

Comment:
Default category should remain.
show_macs_col should also have a standard value.